### DISCUSSION TITLE
Make multiple build targets

### DISCUSSION THREAD

**Discussion by mcullenEST** [strimzi/strimzi-kafka-operator#6221] [Ideas]
When building both the source code (java_install) and the docker images (docker_build) together, the following error appears in the docker-images/base directory: 

![image](https://user-images.githubusercontent.com/92791135/149925528-feacee36-8886-4380-895c-102aaa121935.png)

We understand that this build target is not in this Makefile, but we think it could help make the builds become more backwards compatible and simple by allowing these targets to be run together in a single command (as shown below).  

**To Reproduce** 

Run the make command as follows: 

	make MVN_ARGS="-DskipTests -DskipITs " java_install docker_build 

**Expected behavior** 
 For docker images to be generated

**Comment by scholzj** [comment]
I'm not sure what do you mean with backwards compatibility. I'm not aware of the Java targets ever being in the container builds. But I think this could be easily solved by adding some dummy targets to the container Makefiles.

**Reply by mcullenEST** [reply]
As the 0.26.0 images could be created using one make command (0.27.x requires two seperate commands, as above). It would allow for a single make command to be used to build either of these versions. Yes, it could be resolved with that small fix.
